TestFoundation/TestNSDateFormatter.swift

```diff
@@ -277,5 +278,17 @@ class TestNSDateFormatter: XCTestCase {
         XCTAssertEqual(f.string(from: testDate), "11-03-2016")
         
     }
-    
+
+    func test_setLocalizedDateFormatFromTemplate() {
+        let locale = Locale(identifier: DEFAULT_LOCALE)
+        let template = "EEEE MMMM d y hhmmss a zzzz"
+
+        let f = DateFormatter()
+        f.locale = locale
+        f.setLocalizedDateFormatFromTemplate(template)
```

Comment:
I will write here because it seems the correct place, I'm sorry for all the people who are working on Foundation and on every other project around the world to have used the term "guys" for reference the group working on Foundation. There are many words to indicate a group of people (like, you know "people") and I'm very sorry to have choose and write the less inclusive of all, I would like to apologise to every person who have read my comment. Please forgive me, and keep up with your amazing work.